src/frontend/js/custom-leaflet.js

```diff
@@ -183,7 +222,7 @@ function main_map_init (map, options) {
                 var marker;
 
                 // adjust marker according waypoints
-                if (wp.name == "santropol") {
+                if (wp.santro && !wp.hideMarker) {
                     // add awesome marker for santropol
                     marker =  L.marker([45.516564, -73.575145], {
                         draggable: false,
```

Comment:
This was explicitly added in the file in several locations, but I am also not sure if it is really necessary. In disabling draggable markers one can still create a custom itinerary by dragging the lines between markers.